types/Actions.d.ts

```diff
@@ -80,10 +83,16 @@ declare module 'grafx-studio-actions' {
     }
 
     /**
-     * Represents a Frame inside Actions
+     * All Studio objects that have a name will implement this interface.
      */
-    interface Frame {
+    export interface HasName {
```

Comment:
`NamedObject` or maybe `ObjectWithName` idk, something more specific might be a tiny bit better

Reply:
So far I am in favor of not changing it. I will review this again in the scope of https://github.com/chili-publish/studio-sdk/pull/328